src/tap/test/gcd_nangate45.ok

```diff
@@ -7,6 +7,7 @@
 [INFO ODB-0226] Finished LEF file:  Nangate45/Nangate45_stdcell.lef
 [INFO ODB-0128] Design: gcd
 [INFO ODB-0130]     Created 54 pins.
-[INFO TAP-0004] Inserted 114 endcaps.
+[INFO TAP-0002] Added 0 corner cells
+[INFO TAP-0003] Added 114 boundary cells
 [INFO TAP-0005] Inserted 118 tapcells.
```

Comment:
Is there a reason we are calling them boundary cells rather than endcaps now?  Also we have a mix of added/inserted - let's settle on one.

Reply:
since Inserted was used before, I'll switch to that and switch to endcaps to be consistent